intro-HPC/ch_troubleshooting.tex

```diff
@@ -93,15 +93,15 @@ \section{Issues connecting to login node}
 
 \ifwindows
 
-If you're using PuTTY and get this errormessage:
+If you are using PuTTY and get this error message:
 
 \begin{prompt}
 server unexpectedly closed network connection
 \end{prompt}
 
-it can be because your PuTTY version is too old. Make sure you're using at least
-version 0.70. If your PuTTY version is too old, you can uninstall it and then
-install the latest version. See \autoref{sec:install-putty} for how to do this.
+it is possible that the PuTTY version you are using is too old and doesn't
+support some required (security-related) features.
+If your PuTTY version is too old, you need to use the latest version. See \autoref{sec:install-putty} for how to do this.
```

Comment:
Change this too `Make sure you are using the latest PuTTY version if you are encountering problems connecting (see \autoref{sec:install-putty}). If that doesn't help, please contact \hpcinfo.`